flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -406,6 +414,13 @@ ANY, and(logical(LogicalTypeRoot.BOOLEAN), LITERAL)
     public static final BuiltInFunctionDefinition IF =
             BuiltInFunctionDefinition.newBuilder()
                     .name("ifThenElse")
+                    .callSyntax(
+                            (sqlName, operands) ->
+                                    String.format(
+                                            "IF %s THEN %s ELSE %s",
```

Comment:
is that valid SQL?

Reply:
Why not `asSerializableOperand`?

Reply:
what happens during nesting multiple ifs?

Reply:
> is that valid SQL?

It was wrong. Fixed it.